test/tests/find-alias-test.js

```diff
@@ -0,0 +1,36 @@
+const { expect } = require('chai');
+const proxyquire = require('proxyquire');
+
+
+const ALIASES = {
+    'browser1': {
+        nameRe: /browser1/,
+        cmd:    '-a -b'
+    },
+
+    'browser2': {
+        cmd: '-c -d'
+    }
+};
+
+describe.only('[Utils] Find alias', () => {
```

Comment:
Remove the `.only` hook.
Add the `eslint-plugin-no-only-tests` module and an appropriate eslint rule to avoid this issue in the future.